runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/state/StateRequestHandlers.java

```diff
@@ -137,6 +143,12 @@ static SideInputHandlerFactory unsupported() {
 
     /** Clears the bag user state for the given key and window. */
     void clear(K key, W window);
+
+    /** Returns the currently valid cache token. */
```

Comment:
```suggestion
    /** Returns the currently valid cache token. There is no valid cache token if null is returned. */
```

Reply:
nit: We might want to go with an Iterable here or Optional instead of using null to mark missing cache tokens.

Reply:
Agree, for now `Optional` looks like the best choice.